File: android/api-tests/src/test/java/com/revenuecat/apitests/java/TransactionMapperApiTests.java
package com.revenuecat.apitests.java.mappers;

import com.revenuecat.purchases.hybridcommon.mappers.TransactionMapperKt;
import com.revenuecat.purchases.models.Transaction;

import java.util.Map;

@SuppressWarnings("unused")
class TransactionMapperApiTests {
    private void checkMap(Transaction transaction) {
        Map<String, Object> map = TransactionMapperKt.map(transaction);
    }
}

File: android/api-tests/src/test/java/com/revenuecat/apitests/kotlin/mappers/TransactionMapperApiTests.kt
package com.revenuecat.apitests.kotlin.mappers

import com.revenuecat.purchases.hybridcommon.mappers.map
import com.revenuecat.purchases.models.Transaction

@Suppress("unused", "UNUSED_VARIABLE")
private class TransactionMapperApiTests {
    fun checkMap(transaction: Transaction) {
        val map: Map<String, Any?> = transaction.map()
    }
}
